src/app/page.tsx

```diff
@@ -0,0 +1,339 @@
+                          <label className="block mb-2 text-sm font-medium text-gray-900 absolute z-10 bottom-[1.8rem] bg-white left-4 px-2">Billing address</label>
+                          <input type="email" name="email" id="email" className="w-full border border-gray-300 text-black text-start p-[0.7rem] px-8 rounded-lg" placeholder="name@company.com" />
+                        </div>
+                        <div className="flex justify-between w-full mt-6">
+                          <div className="relative w-[48%]">
+                            <label className="block absolute px-2 text-black bottom-[2.7rem] z-10 bg-white left-4 text-sm font-medium">Adults passengers</label>
+                            <select id="countries" className="border border-gray-300 text-gray-900 text-sm rounded-lg block w-full p-4 ">
+                              <option selected>1</option>
+                              <option value="US">2</option>
+                              <option value="CA">3</option>
+                              <option value="FR">4</option>
+                              <option value="DE">5</option>
+                            </select>
+                          </div>
+                          <div className="relative w-[48%]">
+                            <label className="block px-2 absolute text-black bottom-[2.7rem] z-10 bg-white left-4 text-sm font-medium">Kids passengers</label>
+                            <select id="countries" className="border border-gray-300 text-gray-900 text-sm rounded-lg  block w-full p-4">
+                              <option selected>1</option>
+                              <option value="US">2</option>
+                              <option value="CA">3</option>
+                              <option value="FR">4</option>
+                              <option value="DE">5</option>
+                            </select>
+                          </div>
+                        </div>
+                        <div className="mt-6 text-black flex">
+                          <div>
+                            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill="none">
+                              <path d="M6.5 11.5H8M9.5 11.5H8M8 11.5V7.5H6.5M8.5 5C8.5 5.27614 8.27614 5.5 8 5.5C7.72386 5.5 7.5 5.27614 7.5 5C7.5 4.72386 7.72386 4.5 8 4.5C8.27614 4.5 8.5 4.72386 8.5 5Z" stroke="black" stroke-linecap="round" stroke-linejoin="round" />
+                              <circle cx="8" cy="8" r="6.5" stroke="black" />
+                            </svg>
+                          </div>
+                          <span className="text-sm ml-2">We require to know this information to prepare the lifevest
+                            and other equipment accordingly</span>
+                        </div>
+                      </form>
+                    </div>
+                  </div>
+                </div>
+
+                {/* Second form */}
```

Comment:
Each form can be in his own high-level components.
In this case will be interesting to save the data in a reducer 
(And both components will feel much more simple)

pages/booking/index
pages/booking/partial/booking-form-1
pages/booking/partial/booking-form-2